packages/react-components/src/organisms/RelatedResearchCard.tsx

```diff
@@ -99,14 +84,35 @@ const showMoreStyles = css({
 
 const titleStyles = css({ fontWeight: 'bold', color: charcoal.rgb });
 
+const iconStyles = css({
+  verticalAlign: 'middle',
+  display: 'inline-block',
+  height: `${24 / perRem}em`,
+  marginRight: `${9 / perRem}em`,
+});
+
 type RelatedResearchCardProp = {
   description: string;
-  relatedResearch: EventResponse['relatedResearch'];
+  relatedResearch:
+    | EventResponse['relatedResearch']
+    | gp2.OutputResponse['relatedOutputs'];
+  title?: string;
+  getIconForDocumentType?: (documentType: string) => EmotionJSX.Element;
+  getSourceIcon?: (source: 'Project' | 'Working Group') => EmotionJSX.Element;
+  tableTitles?: [string, string, string]; // ensuring it has exactly 3 elements
 };
 
 const RelatedResearchCard: React.FC<RelatedResearchCardProp> = ({
   relatedResearch,
   description,
+  getSourceIcon,
+  title = 'Related Research',
+  getIconForDocumentType = getIconForDocumentTypeCRN,
+  tableTitles = [
+    'Document Type',
+    'Shared Output Name',
+    'Team or Working Group',
```

Comment:
I'm a bit confused by this > 

The title is "Team or Working Group" - but on line 101 we can have Project or Working Group

Reply:
I couldn't see it anywhere, but are we mixing things. i.e. GP2 has Project and Working Group but CRN has Team and Working Group?

Reply:
This is
```
[
    'Document Type',
    'Shared Output Name',
    'Team or Working Group',
]
```

the default when I'm not passing any `tableTitles` to the component :) I'm passing the appropriate titles for GP2 here https://github.com/yldio/asap-hub/blob/GP2HUB-121-create-related-outputs-card/packages/gp2-components/src/templates/OutputDetailPage.tsx#L113

so they look like this for GP2

Mobile | Desktop   
:-------------------------:|:-------------------------:
<img width="250px" src="https://github.com/yldio/asap-hub/assets/16595804/22a7f0a5-e267-4cd2-943d-8442e82e6759">  | <img width="550px" src="https://github.com/yldio/asap-hub/assets/16595804/ec4a5a5e-f4b6-4c19-8ac3-48fb22ffc8f7">